### DISCUSSION TITLE
why "GitHub Actions is currently disabled for this repository"

### DISCUSSION THREAD

**Discussion by hoangsoft90** [community/community#18900] [Actions]
i face this issue "GitHub Actions is currently disabled for this repository". why & how can i re-enable actions on my repo?

**Comment by mAlex28** [comment]
I have not faced with this issue before but I think this official github documentation will help you: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

**Comment by melvincwng** [comment]
Hi @hoangsoft90, you can try contacting GitHub Support to help investigate the issue.

Link: https://support.github.com/

They usually reply pretty quickly (within couple of hours to 1-2 days). 

Hope this helps you.

**Comment by fedekunze** [comment]
Changing the org or repository settings doesn't work and Github support is non-responsive (see https://github.com/github-community/community/discussions/19015#discussioncomment-3016524)
